proto/serverless/instrumentation/tags/v1/error.proto

```diff
@@ -18,12 +18,23 @@ message ErrorTags {
 
         // An unexpected error that caused the application to fail
         ERROR_TYPE_UNCAUGHT = 1;
-       
-        // An error that was reported via the Serverless SDK.
-        // Error that doesn't explicitly fail the application.
+
+        // An error that was reported by user explictly via the Serverless SDK or console.error call
+        // Error doesn't explicitly fail the application.
+        // Multiple errors of this type can be reported during a single application run
+        ERROR_TYPE_CAUGHT_USER = 2;
```

Comment:
Good point, we don't rely on the names in ingest, we do have some data in prod that has the name already parsed into clickhouse, but I don't believe it will be harmful. We can go ahead and change this.